src/js/providers/html5.js

```diff
@@ -744,7 +744,7 @@ define([
 
         this.setContainer = function(containerElement) {
             _container = containerElement;
-            containerElement.appendChild(_videotag);
+            containerElement.insertBefore(containerElement.firstChild, _videotag);
```

Comment:
only `insertBefore` if there is a `containerElement.firstChild`, otherwise append. Also you got it backwards - video first then firstChild - BTW are you testing this @pajong.